esmvaltool/recipes/recipe_kcs.yml

```
@@ -0,0 +1,155 @@
+# ESMValTool
+# recipe_kcs.yml
+---
+documentation:
+  description: >
+    This recipe reproduces the basic steps described in Lenderink 2014.
+    We just do one scenario at a time; maybe extend later.
```

```suggestion
    This recipe reproduces the basic steps described in Lenderink 2014,
    one scenario at a time.
```
please do not leave TODO items lying around